src/system/template-path-hints.md

```diff
@@ -32,6 +32,10 @@ Before using template path hints, add your IP address to the [whitelist]({% lin
 
     - To activate template path hints for the store, set **Enabled Template Path Hints for Storefront** to `Yes`.
 
+    - To enable template path hints for the store with only passing params in url, set **Enable Hints for Storefront with URL Parameter** to `Yes`.
+
+    - To set value for the url param. default value is `magento`.
```

Comment:
How does setting this value change the behavior?

Reply:
@leslietilling this setting helps to change the URL param value. 
for example, if the developer changed the value from magento to lorem, the developer needs to pass the changed value to URL param (local.magento24.com?templatehints=lorem) to display template hints.